### ISSUE TITLE
Can't run dev server, invalid URL scheme

### ISSUE THREAD

**Issue by boul2gom** [nuxt/nuxt#21835]
### Environment

- Operating System: Windows_NT
- Node Version:     v19.1.0
- Nuxt Version:     3.6.1
- Nitro Version:    2.5.1
- Package Manager:  npm@9.3.0
- Builder:          vite
- User Config:      modules, googleFonts
- Runtime Modules:  @nuxtjs/html-validator@1.4.0, @nuxt/devtools@0.6.4, nuxt-purgecss@2.0.0, @nuxtjs/color-mode@3.3.0, @nuxtjs/device@3.1.0, @nuxtjs/robots@3.0.0, @invictus.codes/nuxt-vuetify@0.2.20, @nuxtjs/google-fonts@3.0.1, @nuxtjs/tailwindcss@6.8.0, @nuxtjs/ionic@0.11.0, @nuxt/image@1.0.0-rc.1
- Build Modules:    -

### Reproduction

There is no reproduction except modules described before, because this issue occurs even with new blanks projects

### Describe the bug

When i execute the ``npm run dev`` command, i got the error ``Only URLs with a scheme in: file, data are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'``. This issue occurs also with WSL, and on differents windows machines

### Logs

```shell-script
ERROR  [worker reload] [worker init] Only URLs with a scheme in: file, data are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'                                                       14:15:13  

  at new NodeError (node:internal/errors:393:5)
  at throwIfUnsupportedURLScheme (node:internal/modules/esm/resolve:986:11)
  at defaultResolve (node:internal/modules/esm/resolve:1066:3)
  at nextResolve (node:internal/modules/esm/loader:161:28)
  at ESMLoader.resolve (node:internal/modules/esm/loader:831:30)
  at ESMLoader.getModuleJob (node:internal/modules/esm/loader:413:18)
  at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
  at link (node:internal/modules/esm/module_job:76:36)
```

**Comment by danielroe** [comment]
Would you provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Comment by danielroe** [comment]
No, that's fine. But would you update it so it's more minimal, following the steps linked? Ideally, for example, you would remove everything but the exact code/configuration necessary to reproduce the issue.

**Comment by boul2gom** [comment]
I setup a [minimal reproduction](https://stackblitz.com/edit/github-pdnydu?file=package.json), but Stackblitz run on linux, and the issue occurs only on windows...

**Comment by danielroe** [comment]
It's fine, I can clone + run on a VM.

**Comment by ah-dc** [comment]
This is caused by [robots module](https://github.com/nuxt-modules/robots).

https://github.com/nuxt-modules/robots/blob/4e3b2d316a9aafebf4e0f7693d1fc603940636da/src/module.ts#L44

The alias should be wrapped with `url.pathToFileURL` to work on windows

**Comment by chaos-zhu** [comment]
me too...

**Comment by Flo0806** [comment]
@danielroe - No longer reproducible - path is now resolved correctly.

### packages/kit/test-21835.js
```typescript
const rawWindowsPath = 'c:/Users/trey/Desktop/proj/.nuxt/robots.mjs'

// Old way
try {
  await import(rawWindowsPath)
} catch (e) {
  console.log('RAW path       =>', e.message.split('\n')[0])
}

// New way
try {
  await import(pathToFileURL(rawWindowsPath).href)
} catch (e) {
  console.log('pathToFileURL  =>', e.code, '(scheme OK now, just file-not-found)')
}
```

**Comment by danielroe** [comment]
nuxtathon closed @Flo0806
